m4/ax_recursive_eval.m4

```diff
@@ -0,0 +1,56 @@
+# ===========================================================================
+#    https://www.gnu.org/software/autoconf-archive/ax_recursive_eval.html
+# ===========================================================================
```

Comment:
any particular reason this commit doesn't replace the licensing with the spx first line while the second commit does?

Reply:
I wanted to keep the import 1:1 to the source from the GNU autoconf archive before changing it, so it's visible what exactly changed